### PR TITLE
🏃Align Makefile with other CAP* repos

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Comment by vincepri** [comment]
/hold

until #256 merges

File: Makefile [hunk 1]
```diff
@@ -192,17 +192,39 @@ release-manifests: $(RELEASE_DIR) ## Builds the manifests to publish with a rele
...
 .PHONY: clean-release
 clean-release: ## Remove the release folder
 	rm -rf $(RELEASE_DIR)
+
+## --------------------------------------
+## Others / Utilities
+## --------------------------------------
+
+.PHONY: run
+run: generate fmt vet ## Run against the configured Kubernetes cluster in ~/.kube/config
```
Comment (chuckha) [inline Makefile]: it looks like fmt and vet get removed in this change but are now referenced here

Comment (vincepri) [reply Makefile]: Yeah those are part of golint, I'll remove these

**Comment by chuckha** [comment]
/hold

looks like fmt and vet snuck back in but they are still deleted?

File: Makefile (hunk 1 above)
Comment (chuckha) [reply Makefile]: these were gone but they look to be back?

**Pull request merged**